website/blog/2023-03-30-relay-15.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Relay v15.0 and VSCode Extension v2.0
+author: The Relay Team
+hide_table_of_contents: false
+---
+
+The Relay team is happy to announce the release of Relay v15 as well as v2 of the Relay VSCode extension. These releases are a long time coming and include lots of new feature, bug fixes and quality of life improvements. While this release is a major version bump and includes a couple breaking changes, few users should be affected and upgrading should be seamless for most use cases. You can find the full list of changes in the [v15 Release Notes](https://github.com/facebook/relay/releases/tag/v15.0.0).
+
+## What's new in Relay 15?
+
+### Support `@refetchable` on interfaces
+
+Previously it wasn't possible to add the `@refetchable` directive to fragments on server interface types. That support has now been added and will no longer error in the Relay compiler.
```

Comment:
> Previously, the `@refetchable` directive wasn't supported on fragments on server interface types.

I don't think the second sentence is necessary. Maybe "on fragment definitions" is a bit clearer, not sure.